domain/schema/secret.go

```diff
@@ -273,19 +286,15 @@ CREATE UNIQUE INDEX idx_secret_unit_consumer_label ON secret_unit_consumer (labe
 
 CREATE TABLE
     secret_remote_unit_consumer (
-        uuid TEXT PRIMARY KEY,
         secret_id TEXT NOT NULL,
-        unit_uuid TEXT NOT NULL,
+        unit_id TEXT NOT NULL,
```

Comment:
A comment like line 127 could help to understand why it's not a FK anymore and also mention it's on the offering model side.
I didn't have proper comment, could you help to add a comment to mention the purpose of creating this table? Thanks.

Reply:
Added comments